### PR TITLE
docs: update Toast to v2 docs

### PR DESCRIPTION
**Pull request by scurker**

### CODE REVIEW

File: docs/pages/components/Toast.mdx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+      name: 'show',
+      type: 'boolean',
+      defaultValue: 'false',
+      description: 'Whether or not to show the toast.',
+    },
+    {
+      name: 'focus',
+      type: 'boolean',
+      defaultValue: 'true',
+      description: 'Whether or not to focus the toast. This will also set the toast to `role="alert"`` when set to true.'
```
Comment (dequejosie) [inline docs/pages/components/Toast.mdx:198]: There is an extra ```

**Comment by dequejosie** [comment]
The link in the dark mode has color contrast issue: Screen Shot 2023-05-04 at 11 10 33 AM

**Comment by dequejosie** [comment]
Not for this particular ticket. The props table looks off for `Toast`. Do we need to adjust the width of docs? 
Screen Shot 2023-05-04 at 11 14 01 AM

**Comment by scurker** [comment]
> The link in the dark mode has color contrast issue: Screen Shot 2023-05-04 at 11 10 33 AM

Good catch! We have a specificity problem here that we need to address.

**Comment by scurker** [comment]
> Not for this particular ticket. The props table looks off for `Toast`. Do we need to adjust the width of docs?
> 
> Screen Shot 2023-05-04 at 11 14 01 AM

Unfortunately the browser decides column widths here and we don't have any utility for setting table column widths. We could probably do something eventually to not break word for the name column / headers.

**Review by dequejosie — DISMISSED** [review]

**Comment by dequejosie** [comment]
Approved. Assuming we are addressing the link issue in another ticket.

**Comment by scurker** [comment]
> Approved. Assuming we are addressing the link issue in another ticket.

I'm going to address it here before I merge.

**Pull request merged**